src/IAM/Domain/User/Model/Email.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Twitter\IAM\Domain\User\Model;
+
+use Twitter\IAM\Domain\User\Model\Exception\InvalidEmailException;
```

Comment:
@adnlv (low) Exception is not a part of the domain model as it has no business behaviour.
```suggestion
use Twitter\IAM\Domain\User\Exception\InvalidEmailException;
```